WcaOnRails/app/views/users/edit.html.erb

```diff
@@ -159,3 +159,17 @@
     <%= f.submit 'Save', class: "btn btn-primary" %>
   <% end %>
 </div>
+
+<script>
+$('.date_picker.form-control').on('dp.error', function(e){
```

Comment:
Oh, I meant every date and datetime input on every page, not every single input everywhere. Presumably people will try typing in poorly formatted dates elsewhere...